packages/url_launcher/url_launcher_android/android/src/main/java/io/flutter/plugins/urllauncher/UrlLauncherPlugin.java

```diff
@@ -61,6 +62,7 @@ public void onAttachedToActivity(@NonNull ActivityPluginBinding binding) {
       return;
     }
 
+    assert urlLauncher != null;
```

Comment:
Is this a new Java thing?

Reply:
Apparently they are from 1.4, so not so new. I wasn't familiar with them myself, but the AS wizard suggested it as a possible fix for a potential NPE warning, and it seemed like the right thing in this case since it shouldn't be possible for the variable to be null due to the check above for a variable that's constructed from it at the same time that it's set.